app/Main.tsx

```
@@ -82,7 +83,13 @@ export default function Home({ posts }) {
         </div>
       )}
       {siteMetadata.newsletter?.provider && (
-        <div className="flex items-center justify-center pt-4">
+        <div className="flex flex-col items-center justify-center pt-4 space-y-2">
```

ESLint

[error] Replace `pt-4·space-y-2` with `space-y-2·pt-4`

(prettier/prettier)